src/PharoLauncher-CLI-Tests/PhLTemplatesCliCommandTest.class.st

```diff
@@ -0,0 +1,84 @@
+Class {
+	#name : #PhLTemplatesCliCommandTest,
+	#superclass : #TestCase,
+	#instVars : [
+		'context'
+	],
+	#category : #'PharoLauncher-CLI-Tests'
+}
+
+{ #category : #adding }
+PhLTemplatesCliCommandTest >> addLocalTemplateNamed: aName [ 
+	^ context launcherModel templateBaseRepository createLocalTemplateNamed: aName
+]
+
+{ #category : #adding }
+PhLTemplatesCliCommandTest >> addTemplateCategoryNamed: aTemplateCategoryName [  
+	| templateCategory |
+	templateCategory := context launcherModel templateBaseRepository newRemoteTemplatesGroupNamed: aTemplateCategoryName.
+	context launcherModel templateBaseRepository setGroups:{ templateCategory }.
+]
+
+{ #category : #adding }
+PhLTemplatesCliCommandTest >> addTemplateNamed: aName inCategory: aCategoryName [ 
+	^ context launcherModel templateBaseRepository createTemplateNamed: aName inCategory: aCategoryName
+]
+
+{ #category : #'process actions' }
+PhLTemplatesCliCommandTest >> assertSuccess [
+	self assert: context exitStatus equals: 0.
+	self assert: self errorString equals: ''
+]
+
+{ #category : #creation }
+PhLTemplatesCliCommandTest >> createOfficialRepositoryInFileSystemWithTemplatesInIt [
```

Comment:
rename #createOfficialRepositoryInFileSystemWithTemplatesInIt to #createOfficialTemplatesCategoryWithSomeTemplates ?